src/pybind/mgr/mirroring/module.py

```diff
@@ -0,0 +1,62 @@
+from typing import List, Optional
+
+from mgr_module import MgrModule, CLIReadCommand, CLIWriteCommand, Option
+
+from .fs.snapshot_mirror import FSSnapshotMirror
+
+class Module(MgrModule):
+    MODULE_OPTIONS: List[Option] = []
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fs_snapshot_mirror = FSSnapshotMirror(self)
+
+    def notify(self, notify_type, notify_id):
+        self.fs_snapshot_mirror.notify(notify_type)
+
+    @CLIWriteCommand('fs snapshot mirror enable')
+    def snapshot_mirror_enable(self,
```

Comment:
nit, might want to add the description of the command using docstring.

Reply:
ack